README.rst

```diff
@@ -22,13 +25,13 @@ OpenWISP 2 monitoring module (Work in progress).
 Available Features
 ------------------
 
-* Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
+* Displays `Uptime, Memory Status, Load Status, Addresses, Neighbours of the **Device** <docs/device_status.png>`_
+* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of `lively charts <#default-metrics>`_
```

Comment:
Uptime is already mentioned in the first bullet point

Reply:
That was referring to the **uptime** shown by Device status tab(data). While second one refers to the Uptime chart. I have added pics for both.

Reply:
For someone who reads it for the first time it's confusing, but we can reword it later